DParser2/Parser/Implementations/DModulesParser.cs

```diff
@@ -311,7 +311,6 @@ public IMetaDeclarationBlock AttributeBlock(DBlockNode module)
 				{
 					switch ((attr as Modifier).Token)
 					{
-						case DTokens.Virtual:
```

Comment:
Dann mach doch den ganzen switch weg^^

Reply:
Was ist denn mit `return`, `scope` und ähnlichem. Gehören die nicht auch hierhin?

Reply:
Nein, denn return und scope (oder vielleicht doch?) können ja nicht außerhalb von funktionsblöcken stehen, oder?

Reply:
Ja, kann sein, dass die nur nach den Funktionsparametern erlaubt sind. Und was ist mit `override` und/oder `abstract`? (Ich hab' mir den Parser aber auch noch nicht genug angeschaut, um die Konstruktion in AttributeBlock zu durchblicken.)